Sources/PackageCollections/Providers/GitHubPackageMetadataProvider.swift

```diff
@@ -61,7 +61,9 @@ struct GitHubPackageMetadataProvider: PackageMetadataProvider {
             // get the main data
             sync.enter()
             let options = self.makeRequestOptions(validResponseCodes: [200])
-            httpClient.get(metadataURL, options: options) { result in
+            var headers = HTTPClientHeaders()
+            headers.add(name: "Accept", value: "application/vnd.github.mercy-preview+json")
```

Comment:
Is this a literal string or something that should be parameterized?

Reply:
are you asking about constants for the header name?